### PR TITLE
correcting screen resolution problem

### PR DESCRIPTION
**Pull request by marcusberger**
Fixes issue #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]

Changes: [Add here what changes were made in this issue and if possible provide links.]

Screenshots for the change:

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -11,6 +11,10 @@ body {
   font-family: "proxima-nova", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Open Sans", "Helvetica Neue", "Arial", sans-serif;
 }
 
+body{
+   overflow-x: hidden;
+}
```
Comment (jenkoian) [inline public/css/main.css]: You can just apply `overflow-x-hidden` on the body element rather than add new styles.

**Comment by jenkoian** [comment]
Can you offer a brief explanation of what this is fixing?

**Comment by marcusberger** [comment]
The screen was overflow

**Comment by jenkoian** [comment]
Closing in prep for 2018 rewrite.

**Pull request closed without merging**